spec/features/comments/user_enters_comments_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe 'User can enter a new comment' do
+  scenario 'user can save a new comment' do
```

Comment:
what happens if the comment isn't saved because they haven't entered valid input? we should have a test for this functionality